README.md

```diff
@@ -10,15 +10,36 @@ Jazz Hands is a automatic build and dev-run tool for Django projects with modern
 * Runs your dev server and automatically rebuilds front-end assets when change are detected!
 * Is Magick :sparkles:!
 
+## Installation
+
+You can install Jazzhands locally in the virtual environment of a Django project.
+
+```
+pip install -e git@github.com:caktus/jazzhands.git
+```
+
 ## Setup
 
-You can install Jazzhands locally in the virtual environment of a Django project. Once installed
+Once installed
 you can install prerequisite NPM packages for ECMAScript compilation with Jazzhand's `setup`
 command.
 
+The setup command needs to know what Babel presets and transform plugins you want. As a basic recommendation you should include the `es2017` preset, which will include all the ES2017 transformations necessary. The setup will add the necessary entries to your `package.json` file as well as generate a `.babelrc` configuration file for you.
```

Comment:
How would I find out what the allowed Babel presets and transform plugins are?  (Maybe link to babel doc or something?)

Reply:
Moved some of this around, along with making the ES2017 preset a default if you omit `-p`, and added the doc link suggested.